src/test/java/nl/tudelft/pixelperfect/event/AsteroidFieldEventTest.java

```diff
@@ -3,6 +3,7 @@
 /**
  * Created by woute on 5/10/2016.
```

Comment:
This javadoc comment should have an author tag and actually describe the class, rather than just "created by".